diff --git a/raylib/Cargo.toml b/raylib/Cargo.toml
index 36b46b3..f7c06e3 100644
--- a/raylib/Cargo.toml
+++ b/raylib/Cargo.toml
@@ -44,89 +44,84 @@ opengl_es_20 = ["raylib-sys/opengl_es_20"]
 
 [[example]]
 name = "specs"
-path = "examples/samples/specs.rs"
+path = "../samples/specs.rs"
 doc-scrape-examples = true
 
 [[example]]
 name = "rgui"
-path = "examples/samples/rgui.rs"
+path = "../samples/rgui.rs"
 doc-scrape-examples = true
 
 [[example]]
 name = "arkanoid"
-path = "examples/samples/arkanoid.rs"
+path = "../samples/arkanoid.rs"
 doc-scrape-examples = true
 
 [[example]]
 name = "logo"
-path = "examples/samples/logo.rs"
+path = "../samples/logo.rs"
 doc-scrape-examples = true
 
 
 [[example]]
 name = "camera2D"
-path = "examples/samples/camera2D.rs"
+path = "../samples/camera2D.rs"
 doc-scrape-examples = true
 
 [[example]]
 name = "raymarch"
-path = "examples/samples/raymarch.rs"
+path = "../samples/raymarch.rs"
 doc-scrape-examples = true
 
 [[example]]
 name = "font"
-path = "examples/samples/font.rs"
+path = "../samples/font.rs"
 doc-scrape-examples = true
 
 [[example]]
 name = "drop"
-path = "examples/samples/drop.rs"
+path = "../samples/drop.rs"
 doc-scrape-examples = true
 
 [[example]]
 name = "texture"
-path = "examples/samples/texture.rs"
+path = "../samples/texture.rs"
 doc-scrape-examples = true
 
 [[example]]
 name = "yaw_pitch_roll"
-path = "examples/samples/yaw_pitch_roll.rs"
+path = "../samples/yaw_pitch_roll.rs"
 doc-scrape-examples = true
 
 [[example]]
 name = "roguelike"
-path = "examples/samples/roguelike.rs"
+path = "../samples/roguelike.rs"
 required-features = ["with_serde"]
 doc-scrape-examples = true
 
 [[example]]
 name = "input"
-path = "examples/samples/input.rs"
-doc-scrape-examples = true
-
-[[example]]
-name = "3d_camera_first_person"
-path = "examples/samples/3d_camera_first_person.rs"
+path = "../samples/input.rs"
 doc-scrape-examples = true
 
 [[example]]
 name = "model_shader"
-path = "examples/samples/model_shader.rs"
+path = "../samples/model_shader.rs"
 doc-scrape-examples = true
 
 [[example]]
 name = "extensions"
-path = "examples/samples/extensions.rs"
+path = "../samples/extensions.rs"
 doc-scrape-examples = true
 
 [[example]]
 name = "asteroids"
-path = "examples/samples/asteroids.rs"
+path = "../samples/asteroids.rs"
 doc-scrape-examples = true
 
 [[example]]
 name = "floppy"
-path = "examples/samples/floppy.rs"
+path = "../samples/floppy.rs"
 doc-scrape-examples = true
 
 [package.metadata.docs.rs]
